components/txn_types/src/types.rs

```diff
@@ -347,10 +346,7 @@ impl OldValue {
 
     pub fn size(&self) -> usize {
         let value_size = match self {
-            OldValue::Value {
-                short_value: Some(v),
-                ..
-            } => v.len(),
+            OldValue::Value { value } => value.len(),
```

Comment:
The size of `ValueTimeStamp` is not zero.

Reply:
`std::mem::size_of::<OldValue>()` computes OldValue enum size, inlcuding `ValueTimeStamp`.